modules/apps/notification/notification-api/src/main/java/com/liferay/notification/constants/NotificationRecipientSettingConstants.java

```diff
@@ -34,6 +34,8 @@ public class NotificationRecipientSettingConstants {
 
 	public static final String NAME_USER_SCREEN_NAME = "userScreenName";
 
+	public static final String NAME_USERS_LANGUAGE = "usersLanguage";
```

Comment:
@pedro-oliveira446 I believe we should rename to `useUserLocale`, see this patterns:

https://github.com/liferay/liferay-portal-ee/blob/0978689ba74c32329bc128bf65fcb56d93140d37/modules/apps/segments/segments-service/src/main/java/com/liferay/segments/model/impl/SegmentsEntryModelImpl.java#L658

https://github.com/liferay/liferay-portal-ee/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/document-library-opener/document-library-opener-service/src/main/java/com/liferay/document/library/opener/service/persistence/impl/DLOpenerFileEntryReferencePersistenceImpl.java#L1018